__tests__/app.test.js

```diff
@@ -1,9 +1,13 @@
 const endpointsJson = require("../endpoints.json");
+const request = require("supertest")
+const app = require("../app")
+// const request = require("supertest")
 /* Set up your test imports here */
 
 /* Set up your beforeEach & afterAll functions here */
 
-describe("GET /api", () => {
+
+describe.only("GET /api", () => {
```

Comment:
Try to avoid leaving .only on when you push, the reason is that a lot of developers (including us!) run software to make sure test suites are passing before code is pushed to avoid broken production code 👍